### ISSUE TITLE
Can't delete account, this is a GDPR violation

### ISSUE THREAD

**Issue by kali4128722545** [bluesky-social/social-app#4963]
**Describe the bug**
I can't delete my account, it says "Rate limit exceeded".

**To Reproduce**

1. Klick "Delete Account"
2. Klick "Send Email"

**Expected behavior**

It should send me a confirmation email, which it doesn't. I is required according to EU regulation, GDPR Article 17 that you are able to delete your account.

**Screenshots**

![Screenshot_2](https://github.com/user-attachments/assets/b87f548b-b3d2-4c1f-a2d6-0bafd1a6e263)

The HTTP Error i get is "Rate Limit Exceeded", which should not happen after clicking it like 10 times a Minute.

**Details**

 - Platform: All
 - Platform version:
 - App version:

**Additional context**

**Comment by kali4128722545** [comment]
This is the Response Header of the Request in Question, if that helps.
![Screenshot_5](https://github.com/user-attachments/assets/5bed8dd4-7379-44aa-80d2-ccf31c3be958)

**Comment by kali4128722545** [comment]
This is the Response Header of the request in question, if that helps.
![Screenshot_8](https://github.com/user-attachments/assets/f15a9203-e533-49c3-ad3f-2adde1d5a590)

**Comment by bnewbold** [comment]
Hi there,
 
It looks like you created a github account to file this one issue. If you need direct support resolving this issue for your account, you can file a support ticket in the Bluesky help center: https://blueskyweb.zendesk.com/hc/en-us

Account deletion is supported. Addressing the rate-limiting technical issue: do you have any additional context about your environment and how you may have hit the rate-limit? Are you on a shared/NAT network? Have you made account deletion requests, or other account-lifecycle requests for multiple accounts from the same device on the same day?

**Comment by kali4128722545** [comment]
Yes, I did, as I didn't receive an email at first. But that was maybe 3
times I tried that, for sure not 15.

Am Mi., 21. Aug. 2024 um 02:54 Uhr schrieb bnewbold <
***@***.***>:

**Comment by kali4128722545** [comment]
And I am deeply sorry but all the support has done so far is treat me as if
I was stupid and can't type in my credentials correctly when i use a
password manager and it all worked correctly just yesterday.

Am Mi., 21. Aug. 2024 um 03:05 Uhr schrieb Nina Wilke <
***@***.***>:

**Comment by kali4128722545** [comment]
I think I will just not use this Platform. I have 2 problems going on
already, one prevents me from using it at all and the other makes it
violate EU law. Sorry man, not a good look.

Am Mi., 21. Aug. 2024 um 03:07 Uhr schrieb Nina Wilke <
***@***.***>:
